.github/workflows/build_wheels_genai_linux_aarch64.yml

```diff
@@ -0,0 +1,66 @@
+name: Build FBGEMM GenAI Aarch64 Linux Wheels
+
+on:
+    pull_request:
+    push:
+      branches:
+        - nightly
+        - main
+        # Release candidate branch look like: v1.11.0-release
+        - v[0-9]+.[0-9]+.[0-9]+-release+
+      tags:
+        # Release candidate tag look like: v1.11.0-rc1
+        - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+        - v[0-9]+.[0-9]+.[0-9]+
+    workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  generate-matrix:
+    if: ${{ github.repository_owner == 'pytorch' }}
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    with:
+      package-type: wheel
+      os: linux-aarch64
+      test-infra-repository: pytorch/test-infra
+      test-infra-ref: main
+      with-cuda: disable
+
+  build:
+    if: ${{ github.repository_owner == 'pytorch' }}
+    needs: generate-matrix
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Nit: There is only one configuration here so you could consider get rid of the `matrix` and set these parameters directly like the x86 workflow